src/eventbridge/target.ts

```diff
@@ -0,0 +1,53 @@
+import { aws_events, aws_events_targets } from "aws-cdk-lib";
+import { EventBus } from "./bus";
+import { Function, isFunction } from "../function";
+import { EventBusRuleInput } from "./types";
+
+export type LambdaTargetProps<P> = {
+  func: Function<P, any>;
+} & Omit<aws_events_targets.LambdaFunctionProps, "event">;
+
+export type EventBusTargetProps<P extends EventBusRuleInput> = {
+  bus: EventBus<P>;
+} & aws_events_targets.EventBusProps;
+
+export type EventBusTargetResource<T extends EventBusRuleInput, P> =
+  | Function<P, any>
+  | LambdaTargetProps<P>
+  | EventBus<T>
+  | EventBusTargetProps<T>;
+
+  /**
+   * Add a target to the run based on the configuration given.
+   */
```

Comment:
formatting - be sure to run prettier

Reply:
ha... it runs on explicit save, I guess I didn't hit save on that file. Good catch.